### PR TITLE
Change MixAmount input type in blend osl shader, code style

### PR DESCRIPTION
**Pull request by usakhelo**
This is complimentary to the 3ds max plugin [PR](https://github.com/appleseedhq/appleseed-max/pull/98).

### CODE REVIEW

File: src/appleseed.shaders/src/max/as_max_blend_material.osl [hunk 1]
```diff
@@ -68,23 +68,23 @@ surface as_max_blend_material
     ClosureOut = BaseMtl;
     
     if (LayerMtl_1)
-        ClosureOut = mix(ClosureOut, LayerMtl_1, (MixColor_1 * MaskColor_1));
+        ClosureOut = mix(ClosureOut, LayerMtl_1, (MixAmount_1 * MaskColor_1));
```
Comment (dictoon) [inline src/appleseed.shaders/src/max/as_max_blend_material.osl]: I suggest removing the mask colors if they are never used.

Comment (usakhelo) [reply src/appleseed.shaders/src/max/as_max_blend_material.osl]: Mask color is the texture, we use them. If no texture then it's 1 so only mix amount affects the mix.

Comment (dictoon) [reply src/appleseed.shaders/src/max/as_max_blend_material.osl]: Oh ok, sorry.

**Comment by dictoon** [comment]
Thank you!

**Pull request merged**